owncloudData/src/main/java/com/owncloud/android/data/user/datasources/implementation/OCRemoteUserDataSource.kt

```diff
@@ -36,4 +39,9 @@ class OCRemoteUserDataSource(
         }.let { return remoteUserInfoMapper.toModel(it)!! }
     }
 
+    override fun getUserQuota(): UserQuota {
```

Comment:
You could use expression body `= whatever` instead of ` { return whatever }` and the same for the `getUserInfo` method